spec/classes/chrony_spec.rb

```diff
@@ -130,6 +133,11 @@
               it { is_expected.to contain_file('/etc/chrony/chrony.keys').with_group('mrt') }
               it { is_expected.to contain_file('/etc/chrony/chrony.keys').with_replace(true) }
               it { is_expected.to contain_file('/etc/chrony/chrony.keys').with_content("0 sunny\n") }
+              if facts[:os]['distro']['release']['major'] > '8'
```

Comment:
why test for rhel 8 ?

Reply:
Well. That was funny.
I noticed on travis it fails, tried this and that.
Today, i cloned repo to tag 0.3.1 made modifications and it was all ok.
So i came to conclusion, 5 days ago i cloned repo with not all things ready (saw you updated a few things) and was modifying not ready code.